# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/workers/ChromeWorker.cpp
# Commit: e7b103c79b1a
# Full Hash: e7b103c79b1aafa9e4f01f4c976e0d2ef46b7c41
# Author: Yulia Startsev <ystartsev@mozilla.com>
# Date: 2022-12-15 09:27:59
# Regressor Bug: 1247687
# File Overlap Count: 1
# Description:
#   Bug 1247687 - Implement requestCredentials for Module Workers; r=jonco
#   
#   This weaves credentials through from the private to script creation.
#   
#   Depends on D155567
# ==============================================================================

diff -r 4dbd510fb042 -r e7b103c79b1a dom/workers/ChromeWorker.cpp
--- a/dom/workers/ChromeWorker.cpp	Wed Dec 14 14:55:36 2022 +0000
+++ b/dom/workers/ChromeWorker.cpp	Wed Dec 14 14:55:36 2022 +0000
@@ -21,7 +21,8 @@
 
   RefPtr<WorkerPrivate> workerPrivate = WorkerPrivate::Constructor(
       cx, aScriptURL, true /* aIsChromeWorker */, WorkerKindDedicated,
-      WorkerType::Classic, u""_ns, VoidCString(), nullptr /*aLoadInfo */, aRv);
+      RequestCredentials::Omit, WorkerType::Classic, u""_ns, VoidCString(),
+      nullptr /*aLoadInfo */, aRv);
   if (NS_WARN_IF(aRv.Failed())) {
     return nullptr;
   }